app/views/install/nhs-england-windows-laptops.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'layout.html' %}
+
+{% block pageTitle %}
+  Installation on an NHS England Windows laptop - NHS.UK prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {{ breadcrumb({
+    items: [
+      {
+        href: "/",
+        text: "Home"
+      }
+    ],
+    href: "/install",
+    text: "Get started"
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <h1 class="nhsuk-heading-l">Installation on an NHS England Windows laptop</h1>
+
+      <p>If you have a Windows laptop issued by NHS England corporate IT, then you can largely follow the <a href="/install/windows/terminal">install guide for Windows</a> but there are some additional steps.</p>
+
+      <h2 class="nhsuk-heading-m">Install software dependencies</h2>
+
+      <p>The following can be installed from the Company Portal application:</p>
```

Comment:
Not sure where this application actually lives? Is it a web-based thing we could link to, or some kind of app buried in the start menu?